Description:
  Neither FreeBSD nor GNU/kFreeBSD have libpcap.  Avoid confusing
  sys/capability.h kernel header with libpcap's
Author: Steven Chamberlain <steven@pyro.eu.org>
Bug-Debian: http://bugs.debian.org/756794
Forwarded: no

--- a/sysdeputil.c
+++ b/sysdeputil.c
@@ -113,6 +113,10 @@
   #define VSF_SYSDEP_HAVE_SETPROCTITLE
 #endif
 
+#if defined(__FreeBSD_kernel__)
+  #undef VSF_SYSDEP_HAVE_LIBCAP
+#endif
+
 #if defined(__NetBSD__)
   #include <stdlib.h>
   #define VSF_SYSDEP_HAVE_SETPROCTITLE
